pyproject.toml

```diff
@@ -1,2 +1,5 @@
 [tool.isort]
 profile = "black"
+
+[tool.ruff]
+lint.ignore = ["E501"]
```

Comment:
Ignore max line lengths for now

Reply:
I quite like the max_length setting :) For python, its the one I find ends up making the code very readable :P let's keep it off for now but would be cool to bring that back some day.